timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -484,15 +493,15 @@ class DirectTabularModel(AbstractMLForecastModel):
         If None, will be determined automatically based on the frequency of the data.
     differences : List[int], default = []
         Differences to take of the target before computing the features. These are restored at the forecasting step.
-        If None, will be set to ``[seasonal_period]``, where seasonal_period is determined based on the data frequency.
         Defaults to no differencing.
     target_scaler : {"standard", "mean_abs", "min_max", "robust", None}, default = "mean_abs"
         Scaling applied to each time series. Scaling is applied after differencing.
-    tabular_hyperparameters : Dict[Dict[str, Any]], optional
-        Hyperparameters dictionary passed to ``TabularPredictor.fit``. Contains the names of models that should be fit.
-        Defaults to ``{"GBM": {}}``.
-    tabular_fit_kwargs : Dict[str, Any], optional
-        Additional keyword arguments passed to ``TabularPredictor.fit``. Defaults to an empty dict.
+    model_name : str, default = "GBM"
+        Name of the tabular regression model. See `autogluon.tabular.registry.ag_model_registry` or
+        `the documentation <https://auto.gluon.ai/stable/api/autogluon.tabular.models.html>`_ for the list of available
+        tabular models.
+    model_hyperparameters : Dict[str, Any], optional
+        Hyperparameters passed to the tabular regression model.
```

Comment:
Parameter names `model_name` and `model_hyperparameters` were chosen for consistency with the covariate regressor
https://github.com/autogluon/autogluon/blob/2c1bb33448a180c49247d4a574b0a40e81af1d2e/timeseries/src/autogluon/timeseries/regressor.py#L43-L47